> Task 4: Hypothesis Test
Over a long period of time, the production of 1,000 high-quality hammers in a factory seems to have reached a
weight with an average of 𝜉11 (in 𝑔) and standard deviation of 𝜉12 (in 𝑔). Propose a model for the weight of the
hammers including a probability distribution for the weight. Provide all the assumptions needed for this model
to hold (even the uncertain ones). What parameters does this model have?
One aims to answer one of the following questions about a new production system:
• (if 𝜉13 = 0): Does the new system make more constant weights?
• (if 𝜉13 = 1): Does the new system make lower weights?
• (if 𝜉13 = 2): Does the new system make higher weights?
• (if 𝜉13 = 3): Does the new system make less constant weights?
To answer this question, a random sample of newly produced hammers is evaluated yielding the weights in 𝜉14.
What hypotheses can you propose to test the question? What test and decision rule can you make to estimate
whether the new system answers the given question? Express the decision rules as logical statements involving
critical values. What error probabilities can you suggest and calculate? Perform the test and draw the conclusion
to answer the question.Also provide the links of references and sources for assignment

Suggest a model for the weight of hammers in terms of:

The assumption is that their weights have a normal distribution, ( N(\mu, \sigma^2) ), where ( \mu ) is the average weight and ( \sigma^2 is variance.

Parameters: ( \mu ) (mean of weight), ( \sigma ) (standard deviation of weight).

State Assumptions:

Weights of hammers follow normal distribution.

A sample of weights was selected randomly.

The process had the same efficiency over time.

Hypotheses Formulation for Each Question:

Is this new system producing more constant weights? For ( ξ_{13} = 0)

Null Hypothesis (( H_0 )): σ_new² ≥σ_old²

Alternative Hypothesis (( H_1 )): σ_new² < σ_old²

Is this new system giving out lower weights? For ( ξ_{13} = 1)

Null Hypothesis (( H_0 )): μ_new ≤μ_old

Alternative Hypothesis (( H_1 )): μ_new> μ_old

Is this new system giving out higher weights? For ( ξ_{13} = 2)

Null Hypothesis (( H_0 )): μ_new ≤μ_old

Alternative Hypothesis (( H_1 )): μ_new> μ_old

Does the new system make less constant weights? For ( ξ_{13} = 3)

Null Hypothesis (( H_0 )): σ_new² ≤σ_old²

Alternative Hypothesis (( H_1 )): σ_new² >σ_old²

Select Test and Decision Rule:

For comparing variances (questions 0 and 3), use the F-test.

For comparing means (questions 1 and 2), use the t-test.

The decision rules involve comparing the test statistic with critical values of respective distributions.

Error Probabilities:

Type I error (( \alpha )): Rejecting a null hypothesis which is true.

Type II error (( \beta )): Not rejecting a null hypothesis which is false.

In most cases, ( \alpha ) is set at 0.05.

Perform Test and Conclude:

Collect sample data ( ξ_{14} ).

Calculate the test statistics.

Compare with critical values to make a decision.

Let us assume that we have the following data on new production system (sample weights in grams):

new_weights = [1010, 1005, 995, 1000, 1003, 1007, 1002, 1008, 1001, 1004]

For each scenario we will perform hypothesis testing. Let's start by defining parameters for old system and new sample data

In [1]:
import numpy as np
from scipy import stats

# Old system parameters
mu_old = 1000  # mean weight in grams
sigma_old = 10  # standard deviation in grams

# New system sample weights
new_weights = [1010, 1005, 995, 1000, 1003, 1007, 1002, 1008, 1001, 1004]

# Calculate sample statistics for the new system
mu_new = np.mean(new_weights)
sigma_new = np.std(new_weights, ddof=1)  # sample standard deviation

mu_new, sigma_new

(1003.5, 4.352521619066865)

The following are sample statistics for the new system:

Mean weight (( \mu_{\text{new}} )): 1003.5 grams

Sample standard deviation (( \sigma_{\text{new}} )): 4.35 grams

Now let us do hypothesis testing for each case.

( \xi_{13} = 0 ): Does the new system make more constant weights?

Null Hypothesis (( H_0 )): ( \sigma_{\text{new}}^2 \geq \sigma_{\text{old}}^2 )

Alternative Hypothesis (( H_1 )): ( \sigma_{\text{new}}^2 < \sigma_{\text{old}}^2 )

Use an F-test to compare variances.

( \xi_{13} = 1 ): Does the new system make lower weights?

Null Hypothesis (( H_0 )): ( \mu_{\text{new}} > = μ_{{{\rm old}}}  )

Alternative Hypothesis (( H_1 )): ( μ=\left[{{{{{\mu}_{n w }} -{μ o l d } }/{{s n (x)}}}\right] ±.0259\times s n (x))

Use a One-Sample T-Test to Compare Means.

( x i _ {13} = 2 ′; ′′ ″: ″; ’‘ ‴: ’′ : ″… Does the new system make higher weights?

Null Hypothesis (( H_0 )) $$: $ µ≤\mu o l d $

Alternative Hypothesis (( H_1 )) $$: $ µ>\mu o l d $

Use a One-Sample T-Test to Compare Means.

( x i _ {13} = 3 ′; ′′ ″: ″; ’‘ ‴: ’′ : ″… Does the new system make less constant weights?

Null Hypothesis (( H_0 )) $$: $ σ_{\text{new}}^2 ≤σ_{\text{old}}^2 $

Alternative Hypothesis (( H_1 )) $$: $ σ_{\text{new}}^2>σ_{\text{old}}^2 $

Use an F-test to compare variances.



In [2]:
alpha = 0.05

# F-test for variances (questions 0 and 3)
F_statistic = (sigma_old ** 2) / (sigma_new ** 2)
df1 = 999  # degrees of freedom for old system
new_sample_size = len(new_weights)
df2 = new_sample_size - 1  # degrees of freedom for new system

# Critical value for one-tailed F-test
F_critical_0 = stats.f.ppf(alpha, df2, df1)
F_critical_3 = stats.f.ppf(1 - alpha, df2, df1)

# P-value for F-test
p_value_F = 1 - stats.f.cdf(F_statistic, df2, df1)

F_statistic, F_critical_0, F_critical_3, p_value_F

(5.2785923753665696,
 0.36877989127799843,
 1.8892357852354515,
 4.7105002731751e-07)

F-Test Results for Variances (Q0 and Q3)

( \xi_{13} = 0 ): Does it make more constant weights of the new system?

Null Hypothesis (( H_0 )): ( \sigma_{\text{new}}^2 \geq \sigma_{\text{old}}^2 )

Alternative Hypothesis (( H_1 )): ( \sigma_{\text{new}}^2 < \sigma_{\text{old}}^2 )

F-Statistic: 5.279

Critical Value (one-tailed, α=0.05): 0.369

Decision Rule: Reject ( H_0 ) if ( F_{\text{statistic}} < F_{\text{critical}} )

We fail to reject ( H_0 ) since 5.279 > 0.369 . The new system does not make more constant weights.

Next, let's perform the t-tests for questions 1

In [3]:
# One-sample t-test for means (question 1)
t_statistic_1, p_value_t_1 = stats.ttest_1samp(new_weights, mu_old, alternative='less')

# One-sample t-test for means (question 2)
t_statistic_2, p_value_t_2 = stats.ttest_1samp(new_weights, mu_old, alternative='greater')

t_statistic_1, p_value_t_1, t_statistic_2, p_value_t_2

(2.5428872684065342,
 0.984219541925825,
 2.5428872684065342,
 0.015780458074175073)

Results of the t-test for Means (Questions 1 and 2).

( \xi_{13} = 1 ): Does the new system lead to a decrease in weight?

Null Hypothesis (( H_0 )): ( \mu_{\text{new}} \geq \mu_{\text{old}} )

Alternative Hypothesis (( H_1 )): ( \mu_{\text{new}} < \mu_{\text{old}} )

t-statistic: 2.543

p-value: 0.984

Decision Rule: Reject ( H_0 ) if p-value < α

Since ( 0.984 > 0.05 ), we fail to reject ( H_0 ). The new system does not make lower weights.

( ξ13 = 2 ∶) does the new system produce heavy weights?

Null Hypothesis (( H_0 ∶)) ( μnew ≤μold )

Alternative Hypothesis (( H_1 ∶)) ( μnew >μold.)

t-statistic: 2.543

p-value: .016

Decision rule : Rejecting null hypothesis if p value is smaller than alpha.

As .016 is less than .05, we reject Ho and infer that the new system yields more weights.

Summary of Conclusions:

( ξ13 = 0 ∶)The new system does not make more constant weights.

( ξ13 = 1 ∶)The new system does not make lower weights.

( ξ13 = 2 ∶)The new system makes higher weights.

( ξ13 =3∶)The new system makes less constant weights.

These have been based on the sample data presented and the hypothesis tests carried out.